packages/core/src/system.ts

```diff
@@ -0,0 +1,50 @@
+import { ActorRef, ActorSystem, ActorSystemInfo } from './types.js';
+
+const children = new Map<string, ActorRef<any>>();
+
+let sessionIdIndex = 0;
+
+export interface Registry {
```

Comment:
What is Registry used for now? shouldn't we just remove this concept in favor of the system?

Reply:
Yes, this can now be removed